config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
-  root 'pages#home'
-  get 'pages/home'
+  root 'pages#homepage'
+  get 'pages/homepage', to: 'pages#homepage'
```

Comment:
all of the routes are also covered in #26 and they look more like what you are suggesting Lee